pex/pex_builder.py

```diff
@@ -112,7 +112,7 @@ def __maybe_run_venv__(pex, pex_root, pex_path):
   sys.exit(2)
 
 __installed_from__ = os.environ.pop(__INSTALLED_FROM__, None)
-sys.argv[0] = __installed_from__ or sys.argv[0]
+sys.argv[0] = os.path.realpath(__installed_from__ or sys.argv[0])
```

Comment:
This breaks the conscript use case of behaving differently based on symlink name.

Reply:
Probably the cause of #2006